### DISCUSSION TITLE
How might I build a function that alters an input expression?

### DISCUSSION THREAD

**Discussion by mattroos** [sympy/sympy#21541] [Q&A] [answered]
I'm brand new to sympy and to computer algebra systems. I'd like to create a function that can take in an expanded expression and return an expression in which powers are replaced by 1. For example, if the function is named ```unpower```:
```
>>> unpower(x*y**3 + x**2)
x*y + x
```
__Is it possible to do this?__ If so, __where might I start in the documentation in order to learn how to do it?__ Tips/advice welcome!

**Comment by oscarbenjamin** [comment, marked as answer]
I suggest reading this as a starting point:
https://docs.sympy.org/latest/tutorial/manipulation.html

**Reply by mattroos** [reply]
Thanks, @oscarbenjamin!

**Reply by mattroos** [reply]
For the benefit of anyone else that might view this question, I got a great answer, with code, [on stackoverflow](https://stackoverflow.com/questions/67741196/in-sympy-how-might-i-build-a-function-that-alters-an-input-expression).
